examples/test.py

```diff
@@ -0,0 +1,25 @@
+import time
+from data_loader import WebdatasetReader
+
+TARS = "pipe: aws s3 cp s3://s-laion/acav100m/mp4_acav100m/{00000..00000}.tar -"
```

Comment:
webdataset parallelism works at the tar level, so you need to pass at least num_prepro_workers tars